module/icp/algs/aes/aes_impl.c

```diff
@@ -413,6 +414,7 @@ icp_aes_impl_set(const char *val, zfs_kernel_param_t *kp)
 	return (aes_impl_set(val));
 }
 
+#if defined(__linux__) || defined(__FreeBSD__)
```

Comment:
These are in here as they have a strcpy() where I can't see the size to us, as it is part of the module_set feature of Linux. Taking the whole module_set capability out for apple.